graylog2-web-interface/src/components/search/SearchResult.jsx

```diff
@@ -144,6 +144,12 @@ const SearchResult = React.createClass({
                          permissions={this.props.permissions} searchInStream={this.props.searchInStream} />
       );
     }
+
+    let loadingIndicator;
+    if (this.props.loadingSearch) {
+      loadingIndicator = <div className="loading-indicator"><Spinner text="Updating search results..."/></div>;
```

Comment:
I think it makes sense to extract this into a separate component straight away, that we can reuse in different places too, including moving its styling from the global stylesheet to a local one.